### DISCUSSION TITLE
[Actions] Scroll to error is too slow

### DISCUSSION THREAD

**Discussion by VoltrexKeyva** [community/community#36509] [Mobile]
Recently [GitHub Actions were introduced to GitHub Mobile](https://github.blog/changelog/2022-10-04-introducing-actions-on-github-mobile/), and now it also scrolls to the error in a workflow if any is present.

However the scrolling speed is very slow compared to the one on the website, this is a problem in workflows that output thousands of lines to the workflow output, an example workflow with this problem would be [this](https://github.com/nodejs/node/actions/runs/3267174071/jobs/5371992428#step:6:1). It takes ages for it to scroll down to the error.

**Comment by iskae** [comment]
Hi @VoltrexMaster 👋 

Thanks so much for reaching out ❤️ 

You are absolutely right and we were able to reproduce the issue 💯 I have created a bug issue for this and we will be fixing it in the upcoming releases. Please stay tuned and please keep sending any issues that you came across 🙇 

Cheers!

**Comment by ossanna16** [comment]
Hey there @VoltrexMaster  and thank you for your feedback! I'm glad you found our community 🙂
